spec/aws/session_store/dynamo_db/table_spec.rb

```diff
@@ -23,7 +23,13 @@ module DynamoDB
       describe Table do
         context 'Mock Table Methods Tests', integration: true do
           let(:table_name) { "sessionstore-integration-test-#{Time.now.to_i}" }
-          let(:options) { { table_name: table_name } }
+          let(:dynamo_db_client) {
+            options = {
+              endpoint: 'http://localhost:8000'
```

Comment:
We should add a section in the README mentioning the use of dynamodb local for integ testing.